browser/resources/settings/brave_overrides/system_page.ts

```diff
@@ -3,22 +3,81 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-import {html, RegisterPolymerTemplateModifications} from 'chrome://resources/brave/polymer_overriding.js'
-import {loadTimeData} from '../i18n_setup.js'
+import {
+  html,
+  RegisterPolymerTemplateModifications,
+  RegisterPolymerComponentBehaviors
+} from 'chrome://resources/brave/polymer_overriding.js'
+import { loadTimeData } from '../i18n_setup.js'
 import '../brave_system_page/brave_performance_page.js'
+import '../shortcuts_page/shortcuts_page.js'
+import { Router } from '../router.js'
+
+RegisterPolymerComponentBehaviors({
+  'settings-system-page': [
+    {
+      onShortcutsClicked_: () => {
+        const router = Router.getInstance()
+        router.navigateTo((router.getRoutes() as any).SHORTCUTS)
+      }
+    }
+  ]
+})
 
 RegisterPolymerTemplateModifications({
   'settings-system-page': (templateContent) => {
+    if (loadTimeData.getBoolean('areShortcutsSupported')) {
+      // Get all of the non-style children - we want to move these into the
```

Comment:
nit: I needed perhaps an additional introductory sentence in the comment here, something like:
```
// Chromium's system section only has a root section but we want to add a subpage for shortcuts.
```